rootston/keyboard.c

```diff
@@ -133,6 +134,22 @@ static void keyboard_binding_execute(struct roots_keyboard *keyboard,
 		wl_list_for_each(output, &keyboard->input->server->desktop->outputs, link) {
 			wlr_output_enable(output->wlr_output, outputs_enabled);
 		}
+	} else if (strcmp(command, "next_layout") == 0) {
+		struct roots_desktop *desktop = keyboard->input->server->desktop;
+		struct wl_list *next = desktop->layout->current_config->link.next;
+
+		if (next == &desktop->config->layouts) {
+			next = next->next;
+		}
+
+		struct roots_layout_config *config =
+			wl_container_of(next, desktop->layout->current_config, link);
+
+		wlr_log(L_DEBUG, "Switching to layout %s", config->name);
```

Comment:
I'm not a fan of different output layouts for rootston.

Reply:
It's more of a development feature. It makes it easy to test multiple layouts without having to restart rootston and helps to test situations where the layout is changing.

Reply:
I'm not a big fan of it but also not opposed to it, but either way it belongs in a separate PR imo

Reply:
Just this specific part of the rootston implementation or the whole rootston implementation?

Reply:
The multiple layouts and switching at runtime part